lib/salus/scanners/pattern_search.rb

```diff
@@ -96,6 +97,12 @@ def run
               if match['required']
                 failure_messages << "Required pattern \"#{match['regex']}\" was not found " \
                   "- #{match['message']}"
+                all_misses << {
```

Comment:
This looks good.  Question on the all hits block above... I see that we are passing sift the -n (show line numbers).  As we update for SARIF we could consider including a field on the hits for the line of code information (or two to include the filename too).  Thoughts?